index.html

```diff
@@ -14802,6 +14802,29 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
               <pref>aria-owns</pref> element references in order to build a consistent model of the content.
             </p>
             <p>Authors MUST NOT specify <pref>aria-owns</pref> on an element which has <a href="#childrenArePresentational">Presentational Children</a>.</p>
+            <p><pref>aria-owns</pref> is resolved in the order it is encountered in the <abbr title="Document Object Model">DOM</abbr>. Every element referenced by <pref>aria-owns</pref> will determine its <a href="#tree_exclusion">exposure to the accessibility tree</a> after its change in ownership is resolved. However:</p>
+            <ul>
+              <li>User agents MUST NOT resolve <pref>aria-owns</pref> when it is set on an element that has been <a href="#tree_exclusion">excluded from the accessibility tree</a>.</li>
+              <li>User agents MUST NOT resolve <pref>aria-owns</pref> when it references an element that is, or has a DOM ancestor that is, [=element/hidden from all users=].</li>
+            </ul>
+            <p>In the following example, “(opens in a new window)” is <a href="#tree_inclusion">included in the accessibility tree</a> by virtue of its changed ownership.</p>
+            <pre class="example highlight">
+&lt;a href="https://www.w3.org/" target="_blank" aria-owns="new-window-warning"&gt;
+  World Wide Web Consortium
+&lt;/a&gt;
+&lt;div aria-hidden="true"&gt;
+  &lt;span id="new-window-warning"&gt; (opens in a new window)&lt;/span&gt;
+&lt;/div&gt;
+            </pre>
+            <p>In the following example, “(opens in a new window)” remains <a href="#tree_exclusion">excluded from the accessibility tree</a> since its <abbr title="Document Object Model">DOM</abbr> ancestor is [=element/hidden from all users=] in host language terms.</p>
+            <pre class="example highlight">
+&lt;a href="https://www.w3.org/" aria-owns="new-window-warning"&gt;
+  World Wide Web Consortium
+&lt;/a&gt;
+&lt;div hidden&gt;
+  &lt;span id="new-window-warning"&gt; (opens in a new window)&lt;/span&gt;
+&lt;/div&gt;
+            </pre>
```

Comment:
per my comment in the related WPT, I would very much like this final example added.

I think this PR looks good, otherwise.
```suggestion
            </pre>
             <p>In the following example, the <code>&lt;div id="instructions"&gt;</code> and its content remains <a href="#tree_inclusion">included in the accessibility tree</a> 
		since the element referencing it with the <code>aria-owns</code> attribute is [=element/hidden from all users=].</p>
            <pre class="example highlight">
&lt;div hidden aria-owns="instructions"&gt;
  ...
&lt;/a&gt;
...
&lt;div id="instructions"&gt;
  Instructions go here...
&lt;/div&gt;
            </pre>
```